.github/workflows/main.yml

```diff
@@ -92,3 +92,39 @@ jobs:
       # The Windows CI transforms the `c-library` symlink into a real directory, modifying the Git
       # state, so we ignore these changes with `--allow-dirty` here.
       run: scripts/verify-publish.sh --allow-dirty
+
+  itt_python_build:
```

Comment:
Please combine these two jobs `itt_python_build` and `itt_python_unit_test` into common one with 3 steps inside: 1. code checkout; 2. build/install; 3. test;